test/script/DeployAllContracts.t.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.8.0 <0.9.0;
+
+import {Test} from "forge-std/Test.sol";
+
+import {DeployConstantProduct} from "script/DeployAllContracts.s.sol";
+
+contract DeployAllContractsTest is Test {
```

Comment:
Does this actually inherit both the oracle and the constant product?

Reply:
Fixed! The script `DeployConstantProduct` used to be the one deploying everything. I decided to split off that contract as well but I forgot to change the import.